launcher/ui/MainWindow.cpp

```diff
@@ -2037,6 +2049,80 @@ void MainWindow::on_actionKillInstance_triggered()
     }
 }
 
+#ifdef Q_OS_WIN
+#define WIN32_LEAN_AND_MEAN
+#include "windows.h"
+#include "winnls.h"
+#include "shobjidl.h"
+#include "objbase.h"
+#include "objidl.h"
+#include "shlguid.h"
+#endif
+
+void MainWindow::on_actionCreateInstanceShortcut_triggered()
```

Comment:
there's already some barebones code for shortcut creation in [`FileSystem.cpp:341`](https://github.com/PrismLauncher/PrismLauncher/blob/54c26b05faa73dd6c8eb21f6fa898d438a3e867a/launcher/FileSystem.cpp#L341). Maybe you could use / do this stuff there instead?

Reply:
oh whoops
Nice to know there's a way to do it on Windows _without_ having to do COM stuff, though apparently this path is broken? Will look at it later.

Reply:
...no, wait, I think it's doing the same thing as I'm doing, damn